src/Windowing/Silk.NET.Windowing.Common/Structs/WindowOptions.cs

```diff
@@ -61,7 +61,7 @@ public WindowOptions(ViewOptions opts)
         public int? PreferredStencilBufferBits { get; set; }
 
         /// <inheritdoc />
-        public Vector4D<int>? PreferredBitDepth { get; }
+        public Vector4D<int>? PreferredBitDepth { get; set; }
```

Comment:
Why?

Reply:
Why not?

Reply:
We don't document what these do. What do they do anyways?

Reply:
They indicate your preferred red, green, blue, and alpha bits of the window's OpenGL framebuffer.

Reply:
Well, I'm fine with it then. Not happy, but fine.